macros/AddonSidebar.ejs

```diff
@@ -408,12 +411,14 @@ var text = mdn.localStringMap({
       <details>
         <summary><%=text['Distribution/Publishing']%></summary>
         <ol>
+          <li><a href="<%=baseURL%>Distribution"><%=text['Distribution/Signing']%></a></li>
```

Comment:
I think this is not intended? It just seems to duplicate the previous item.

Reply:
Removed in latest commit